README.md

```diff
@@ -328,6 +328,16 @@ external dependencies. But running `riff run cargo build` succeeds because Riff
 is able to infer which external dependencies are missing and provide them in the
 background using Nix.
 
+## Garbage Collection
+
+While using [Nix] grants Riff its powers, it also comes with some drawbacks,
```

Comment:
First sentence can be more concise. Maybe something like:

One of the disadvantages of Riff using Nix is that Nix can use substantial disk space.